### PR TITLE
Working sticky footer + removed margins, br :tada:

### PR DESCRIPTION
**Pull request by FredFaust**
Also added some CSS (padding) to basic elements like p, ul, ol, h1, h2,
h3

### CODE REVIEW

**Comment by FredFaust** [comment]
@felpel Review plz

File: views/pages/help/disciplines.jade [hunk 1]
```diff
@@ -5,94 +5,94 @@ block content
     
     h2 Le Camouflage
     p 
-    | À la campagne, il peut se cacher parmi les arbres et les rochers et se rendre de cette
-    | façon invisible à l'ennemi même s'il passe tout près de lui. Dans une ville, cette
-    | Discipline donnera à celui qui l'utilise la faculté d'avoir l'air d'un habitant du cru,
-    | tant par l'apparence que par l'accent ou la langue employée. On peut ainsi trouver
-    | un abri où se cacher en toute sécurité.
-    br 
+      | À la campagne, il peut se cacher parmi les ares et les rochers et se rendre de cette
```
Comment (felpel) [inline views/pages/help/disciplines.jade]: arbres*

File: views/pages/help/disciplines.jade [hunk 2]
```diff
@@ -5,94 +5,94 @@ block content
...
+      | façon invisible à l'ennemi même s'il passe tout près de lui. Dans une ville, cette
+      | Discipline donnera à celui qui l'utilise la faculté d'avoir l'air d'un habitant du cru,
+      | tant par l'apparence que par l'accent ou la langue employée. On peut ainsi trouver
+      | un ai où se cacher en toute sécurité.
```
Comment (felpel) [inline views/pages/help/disciplines.jade]: abri*

File: views/pages/help/disciplines.jade [hunk 3]
```diff
@@ -5,94 +5,94 @@ block content
...
+      | cette arme dès le début de votre aventure ; au cours de celle-ci, cependant, 2
+      | l'occasion vous sera donnée de vous la procurer. Vous pourrez d'ailleurs acquérir
+      | diverses armes au cours de votre quête mais vous n'aurez pas le droit d'en posséder
+      | plus de 2 à la fois.
     ol(start='0')
         li Le poignard
         li La lance
         li La masse d'armes
-        li Le sabre
+        li Le sae
```
Comment (felpel) [inline views/pages/help/disciplines.jade]: sabre*

File: views/pages/home.jade [hunk 4]
```diff
@@ -1,56 +1,56 @@
...
         qu'ils ont faite du mage bossu qui a succ&#233;d&#233; sur son tr&#244;ne au f&#233;roce tyran ne peut correspondre qu'&#224; un 
         seul homme : Vonotar le Tra&#238;tre. 
-    br
+    
     p.
-        Apr&#232;s la d&#233;faite des Ma&#238;tres des T&#233;n&#232;bres, Vonotar alla se r&#233;fugier dans 
+        Apr&#232;s la d&#233;faite des Ma&#238;tres des T&#233;n&#232;es, Vonotar alla se r&#233;fugier dans 
         les d&#233;serts glac&#233;s de Kalte. Il parvint jusqu'&#224; la Forteresse de Glace d'Ikaya o&#249; il r&#233;ussit, par la ruse 
-        et la tromperie, &#224; convaincre le cruel Brumalmarc de le prendre &#224; son service comme mage officiel. De la 
+        et la tromperie, &#224; convaincre le cruel umalmarc de le prendre &#224; son service comme mage officiel. De la
```
Comment (felpel) [inline views/pages/home.jade]: Brumalmarc*

File: views/pages/layout.jade [hunk 5]
```diff
@@ -6,15 +6,17 @@ html
...
+          li
+            a(href='/home', class=selectedNav === 'home' ? 'nav-selected' : 'nav-not-selected') Accueil
+          li
+            a(href='/pages/0', class=selectedNav === 'game' ? 'nav-selected' : 'nav-not-selected') Jeu
+          li
+            a(href='/help', class=selectedNav === 'help' ? 'nav-selected' : 'nav-not-selected') Aide
+      block content
+    footer.site-footer()
+      span Laboratoire #2  /  Présenté à M. Konstantinos Lambrou-Latreille  /  Par Jean-Frédéric Faust et Félix Pelletier
+      br
```
Comment (felpel) [inline views/pages/layout.jade]: Remove this ugly br pls

**Comment by felpel** [comment]
Watch carefully my comments please, @FredFaust :-1:

File: views/pages/help/disciplines.jade (hunk 3 above)
Comment (FredFaust) [inline views/pages/help/disciplines.jade]: omg c'est parce que je me pensais cool de faire un Find and Replace... rekt

**Comment by FredFaust** [comment]
All comments are now outdated, merging now

**Pull request merged**